crypto/rsa/rsa_oaep.c

```diff
@@ -259,9 +259,6 @@ int RSA_padding_check_PKCS1_OAEP_mgf1(unsigned char *to, int tlen,
      * To avoid chosen ciphertext attacks, the error message should not
      * reveal which kind of decoding error happened.
      */
-    RSAerr(RSA_F_RSA_PADDING_CHECK_PKCS1_OAEP_MGF1,
```

Comment:
This will break error-reporting for legitimate uses of RSA OAEP. Unlike PKCS1 v1.5 padding, OAEP does not need this fanfare to begin with. The padding check internally needs to be constant-time, but it's fine to report an error after that's all done.